### ISSUE TITLE
Unable to generate from Kubernetes API v1.10.7+

### ISSUE THREAD

**Issue by mingfang** [pulumi/pulumi-kubernetes#183]
Changing https://github.com/pulumi/pulumi-kubernetes/blob/f9345c594528ee94a842ddc3206198a5b76b3ef0/Makefile#L12 to v1.10.7 and v1.11.2 causes the build to fail with this error
```
types/input.ts:736:51 - error TS2694: Namespace '"/go/src/github.com/pulumi/pulumi-kubernetes/pack/nodejs/types/input".apiextensions.v1beta1' has no exported member 'CustomResourceSubresourceStatus'.

736       status?: pulumi.Input<apiextensions.v1beta1.CustomResourceSubresourceStatus>
                                                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


types/output.ts:691:46 - error TS2694: Namespace '"/go/src/github.com/pulumi/pulumi-kubernetes/pack/nodejs/types/output".apiextensions.v1beta1' has no exported member 'CustomResourceSubresourceStatus'.

691       readonly status: apiextensions.v1beta1.CustomResourceSubresourceStatus
                                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

```

**Comment by lukehoban** [comment]
Let's see if we can update to latest in M19, and try to establish an ability to update regularly so that the newest Kubernetes features are always available from Pulumi.

**Comment by joeduffy** [comment]
@lblackstone Something you can look into, to help load balance @hausdorff?

**Comment by lukehoban** [comment]
This is in progress in https://github.com/pulumi/pulumi-kubernetes/pull/293.

**Comment by joeduffy** [comment]
Ah great - did not see that, glad this is landing soon!

**Comment by hausdorff** [comment]
Fixed with #293.
